pandas/tests/dtypes/test_inference.py

```diff
@@ -552,6 +552,15 @@ def test_maybe_convert_objects_datetime(self):
         out = lib.maybe_convert_objects(arr, convert_datetime=1, convert_timedelta=1)
         tm.assert_numpy_array_equal(out, exp)
 
+    def test_maybe_convert_objects_nullable_integer(self):
+        # GH27335
+        arr = np.array([2, np.NaN], dtype=object)
+        result = lib.maybe_convert_objects(arr, convert_to_nullable_integer=1)
+        from pandas.core.arrays import IntegerArray
+
+        exp = IntegerArray(np.array([2, 0], dtype="i8"), np.array([False, True]))
```

Comment:
can you  parameterize this test with an int64 array as well

Reply:
Parameterized as requested